docs/data/material/components/snackbars/snackbars.md

```diff
@@ -87,7 +87,7 @@ This demo shows how to display multiple Snackbars without stacking them by using
 
 {{"demo": "ConsecutiveSnackbars.js"}}
 
-## Third-party integrations
+## Supplementary components
```

Comment:
Seems to match the other sections. It's not even an integration anyway notistack copies the style but doesn't depend on Material UI.